docs/source/manual/snakemake.md

```diff
@@ -1,29 +1,97 @@
+# [Alpha Pre-release] Snakemake Integration
 
-## Technical preview. Currently not ready for production use
+## Pre-release Disclaimer - Currently not ready for production use
 
-Latch support for Snakemake is in active development. Some workflows already work but a lot of common use cases need minor work. New documentation is also in development.
+Latch support for Snakemake is in active development. Some workflows already work but a lot of common use cases need minor work. This documentation is also in active development.
 
-This preview was created to integrate miscelanneous improvements that accumulated over the course of developing the integration and to prevent the codebase from further diverging from the main branch.
+This pre-release was created to integrate miscellaneous improvements that accumulated over the course of developing the integration and to prevent the codebase from further diverging from the main branch.
 
 ## Getting Started
 
-Register using the Latch CLI. Example: `latch register workflow_folder/ --snakefile workflow_folder/src/Snakefile`
+Latch's snakemake integration allows developers to build graphical interfaces to expose their workflows to wet lab teams and managed cloud execution of snakemake jobs.
 
-Modify the Snakefile to make it compatible with cloud execution:
+A primary design goal for integration is to allow developers to register existing projects with minimal added boilerplate and modifications to code. Here we outline exactly what these changes are and why they are needed.
+
+Recall a snakemake project consists of a `Snakefile`, which describes workflow rules using an ["extension"](https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html) of Python, and associated python code imported and called by these rules in the `Snakefile`.To make this project compatible with Latch, we need to do the following:
+
+1. Identify and construct explicit parameters for each file dependency in `latch_metadata.py`
+2. Build a container with all runtime dependencies
+3. Ensure your `Snakefile` is compatible with cloud execution
+
+### Constructing a `latch_metadata.py` file
+
+The snakemake framework was designed to allow developers to both define and execute their workflows. This often means that the workflow parameters are sometimes ill-defined and scattered throughout the project as configuration values, static values in the `Snakefile` or command line flags.
+
+To construct a graphical interface from a snakemake workflow, the parameters need to be explicitly identified and defined so that they can be presented to scientist to be filled out through a web application. The `latch_metadata.py` file holds these parameter definitions, along with any styling or cosmetic modifications the developer wishes to make to each parameter.
+
+Asking the question, "what are the minimum set of files needed to run my workflow to completion" is a good way to begin identifying what these parameters might be. 
```

Comment:
Asking the question unfortunately was not illuminating for me lol

It'd be helpful to give an example of a `config.yaml` file, and how that translates to `latch_metadata.py`